frontend/packages/dev-console/src/components/catalog/WebTerminalConfiguration.tsx

```diff
@@ -0,0 +1,363 @@
+        },
+      },
+      spec: {
+        ...webTerminalExec?.spec,
+        components: getUpdatedComponentTimeout(webTerminalExec?.spec?.components, csTimeout),
+      },
+    };
+
+    const newWebTerminalTooling = {
+      ...webTerminalTooling,
+      metadata: {
+        ...webTerminalTooling?.metadata,
+        annotations: {
+          ...webTerminalTooling?.metadata?.annotations,
+          'web-terminal.redhat.com/unmanaged-state': imageCheckBox ? 'true' : 'false',
+        },
+      },
+      spec: {
+        ...webTerminalTooling?.spec,
+        components: getUpdatedComponentImage(webTerminalTooling?.spec?.components, image),
+      },
+    };
+
+    try {
+      k8sUpdateResource({
+        model: DevWorkspaceTemplateModel,
+        data: newWebTerminalExec,
+        name: 'web-terminal-exec',
+        ns: operatorNamespace || DEFAULT_NS_OPERATORS,
+      });
+      k8sUpdateResource({
+        model: DevWorkspaceTemplateModel,
+        data: newWebTerminalTooling,
+        name: 'web-terminal-tooling',
+        ns: operatorNamespace || DEFAULT_NS_OPERATORS,
+      });
+      fireTelemetryEvent('Console cluster configuration changed', {
+        customize: 'Web Terminal configuration items',
+        timeout: csTimeout,
+        image,
+      });
+      setSaveStatus({ status: 'successful' });
+    } catch (e) {
+      setSaveStatus({ status: 'error', error: e });
+    }
+  };
+  return (
+    <FormSection
+      title={t('devconsole~Web Terminal Configuration')}
+      data-test="web-terminal form-section"
+    >
+      <FormHelperText isHidden={false}>
+        {t('devconsole~As admin you can change the default timeout and image of Web Terminal.')}
+      </FormHelperText>
+      <FormGroup
+        label={t('devconsole~Timeout')}
+        helperText={t('devconsole~Set timeout for the terminal.')}
+      >
+        <div className="pf-c-input-group">
+          <NumberSpinner
+            onChange={onValueChange}
+            changeValueBy={changeValueBy}
+            name="timeout-value"
+            id="timeout-value"
+            data-test="timeout-value"
+            value={value}
+            min={0}
+          />
+          <Dropdown
+            title={TimeoutUnits[unit]}
+            selectedKey={unit}
+            name="timeout-unit"
+            className="request-size-input__unit"
+            items={TimeoutUnits}
+            onChange={onUnitChange}
+            ariaLabel={t('devconsole~Number of {{sizeUnit}}', {
+              sizeUnit: TimeoutUnits[unit],
+            })}
+          />
+        </div>
+      </FormGroup>
+      <FormGroup>
+        <Checkbox
+          id="timeout-value-checkbox"
+          data-test="timeout-value-checkbox"
+          isChecked={timeoutCheckBox}
+          key="timeout-value-checkbox"
+          onChange={() => setTimeoutCheckBox(!timeoutCheckBox)}
+          label={t('devconsole~Keep timeout value even after operator restart or update')}
```

Comment:
Wdyt about making it clear that this changes a resource to "Unmanaged" 

```suggestion
          label={t('console-app~Mark the configuration resource as "Unmanaged" to keep the default timeout even after operator restart or update.')}
```